assets/performanceprofile/configs/99-netdev-physical-rps.rules

```diff
@@ -1 +1 @@
-SUBSYSTEM=="queues", ACTION=="add", ENV{DEVPATH}=="/devices/pci*/queues/rx*", TAG+="systemd", ENV{SYSTEMD_WANTS}="update-rps@%p.service"
\ No newline at end of file
+SUBSYSTEM=="queues", ACTION=="add", ENV{DEVPATH}=="/devices/pci*/queues/rx*", TAG+="systemd", PROGRAM="/bin/systemd-escape --path --template=update-rps@.service $env{DEVPATH}", ENV{SYSTEMD_WANTS}="%c"
```

Comment:
this looks clean and it's probably the cleanest approach, but we're spawning a new process for each invocation. Can we avoid that? Or is it a performance concern at all, actually?

Reply:
Yeah, that is unfortunate, however we could not avoid that. Also we only do that for physical devices, so hopefully just once per boot.

Reply:
fair enough